core/frontend/src/tile/PrimaryTileTree.ts

```diff
@@ -19,8 +18,9 @@ import { GeometricModel3dState, GeometricModelState } from "../ModelState";
 import { RenderClipVolume } from "../render/RenderClipVolume";
 import { RenderScheduleState } from "../RenderScheduleState";
 import { SpatialViewState } from "../SpatialViewState";
+import { System } from "../render/webgl/System";

```

Comment:
Don't import from the webgl directory and don't assume `IModelApp.renderSystem` is-a `System`.
Probably some API is missing from RenderSystem.

Reply:
Done.